connector_onpay/models/hr_employee.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    onpay_id = fields.Char(
```

Comment:
Suggestion: `_id`  usually means a many2one field, and may lead to confusion when reading code.
I would prefer to name this `onpay_code`.